app/interceptors/ExceptionInterceptor.java

```diff
@@ -72,7 +71,7 @@ private int getStatusCode(final Throwable throwable) {
     private String getMessage(final Throwable throwable, final int status) {
         String message = throwable.getMessage();
         if (StringUtils.isBlank(message)) {
-            message = HttpStatus.getStatusText(status);
+            message = Integer.toString(status);
```

Comment:
It's a new dependency to Jetty (owned by Eclipse.org).  The dependency to apache httpcore is transitive.  I didn't check.  It's likely introduced by the AWS client or the Stormpath client.  HttpStatus is still used in our code base.  The version is 4.3.6 not the latest.  But I expect HttpStatus shouldn't change often.